### ISSUE TITLE
Implement do_step in the FMU API

### ISSUE THREAD

**Issue by Maghnie** [RWTH-EBC/ebcpy#38]
As decided from feedback about the unified FMU API interface, a `doStep` function (wrapper) is needed for simulating single steps with FMUs.

Example: https://github.com/CATIA-Systems/FMPy/blob/master/fmpy/examples/custom_input.py

**Comment by konrad234** [comment]
Implementing a do_step function motivates a different class structure (#55)

**Comment by Maghnie** [comment]
Hi @FWuellhorst, does this issue still seem relevant?

From my side, it can be closed.

**Comment by FWuellhorst** [comment]
@Maghnie No, I don't need this function, and I think @FelixStege agrees?

**Comment by FelixStege** [comment]
Yes, I agree. I am closing this Issue.
